models/channel_helpers.go

```diff
@@ -183,6 +116,73 @@ func validateWebHookConfig(c *WebHookConfig) error {
 	return nil
 }
 
+// FindChannels returns saved notification channels configuration.
+func FindChannels(q *reform.Querier) ([]Channel, error) {
+	structs, err := q.SelectAllFrom(notificationChannelTable, "")
```

Comment:
result would be a better name than structs imo

Reply:
I took that naming from https://github.com/percona/pmm-managed/blob/d7bd47a3e97e09a0d1109b77df033231bff811e6/models/agent_helpers.go#L155

However, it seems that there are different names for that in this file. How about name it `rows`?

Reply:
Renamed to `rows`